### PR TITLE
Bringing omnibus process in line with other Chef applications

### PR DESCRIPTION
**Pull request by tyler-ball**
These are some changes I found helpful while building https://github.com/chef/chef-server/pull/1065 locally. I wanted to move them into another PR to clean up that one.

This allowed me to `cd omnibus && bundle install && bundle exec kitchen converge centos-7` to get a good centos builder

### CODE REVIEW

**Comment by dmccown** [comment]
👍

**Comment by dmccown** [comment]
Opps. Wrong button :(.

File: omnibus/Gemfile [hunk 1]
```diff
@@ -1,17 +1,25 @@
-source 'https://rubygems.org'
+source "https://rubygems.org"
 
-gem 'rake'
-gem 'chefspec'
-gem 'berkshelf'
-gem 'bundler', '>1.10'
+gem "omnibus", github: "chef/omnibus", branch: "master"
```
Comment (stevendanna) [inline omnibus/Gemfile]: Does this produce bundler warnings because of the lack of HTTPS?

Comment (tyler-ball) [inline omnibus/Gemfile]: It does! Updating now to use https

**Comment by stevendanna** [comment]
We've removed Gemfile.lock for now.

**Pull request closed without merging**